ts/models/conversation.ts

```diff
@@ -1431,19 +1493,9 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     }
   }
 
-  private async addSingleMessage(messageAttributes: MessageAttributesOptionals) {
+  public async addSingleMessage(messageAttributes: MessageAttributesOptionals) {
```

Comment:
I'd rather keep this private and use the addOutgoingMessage and addIncomingMessage. The typing is a bit more strict with those